lib/logstash/inputs/rethinkdb.rb

```diff
@@ -32,7 +32,9 @@ class LogStash::Inputs::RethinkDB < LogStash::Inputs::Base
   config :backfill, :default => true
   #ssl support
   config :ca_certs, :default => nil
-
+  # Credentials as of RethinkDB v2.3.x
+  config :user, :validate => :string, :default => "admin"
```

Comment:
@AtnNn Ive added the default values, but if there is a way in ruby to leave it open and let rethink choose the defaults It would be better, I'm just not sure what would happen if these are past as Nil in line 62 or if there is a way to conditinaly apply/pass configurations in ruby.